xbstrap/base.py

```diff
@@ -10,14 +10,17 @@
 import shutil
 import stat
 import subprocess
+import sys
 import tarfile
 import tempfile
 import urllib.request
 import zipfile
+from distutils.version import StrictVersion
 from enum import Enum
 
 import colorama
 import jsonschema
+import pkg_resources
```

Comment:
due to this, we now depend on setuptools.  consider instead extracting the version into a file that setup.py reads/imports

Reply:
that we don't declare it...  plus, decently sure this won't work when doing a distro install.  there's simply no need